haskell.el

```diff
@@ -49,6 +49,8 @@
     (define-key map (kbd "C-c C-x") 'haskell-process-cabal)
     (define-key map [?\C-c ?\C-b] 'haskell-interactive-switch)
     (define-key map [?\C-c ?\C-z] 'haskell-interactive-switch)
+    (define-key map (kbd "M-n") 'haskell-goto-next-error)
+    (define-key map (kbd "M-p") 'haskell-goto-prev-error)
```

Comment:
@DanielG, do you mean with:
```
(defvar ghc-previous-key    "\ep")
(defvar ghc-next-key        "\en")
```
I must say I overlooked this, then..

But then, there's the more pressing question of having two sets of overlays on at the same time, which is still unresolved.